examples/magento-graphcms/components/GraphCMS/RowProduct/RowProduct.tsx

```diff
@@ -19,14 +20,15 @@ type VariantRenderer = Record<
 
 type RowProductProps = RowProductFragment & {
   renderer?: Partial<VariantRenderer>
-} & ProductSpecsFragment & { items?: unknown }
+} & ProductSpecsFragment & { items?: unknown } & { sku?: string | null | undefined }
 
 const defaultRenderer: Partial<VariantRenderer> = {
   Specs,
   Backstory,
   Feature,
   FeatureBoxed,
   Grid,
+  Recent,
```

Comment:
You've implemented this with a Row rendered via Hygraph, do you expect this row to be moved around for customers? If not, might it be easier to 'just' add it to certain areas?

_If_ we choose the Hygraph route, please make sure that the Hygraph schema has a correct  migration an the upgrade docs communicate that the hygraph schema needs to be migrated.

Reply:
We offer all PDP content in this manner, so it would make sense to also offer recently viewed products as a RowProduct variant in Hygraph
![Screenshot 2023-10-20 at 13 12 03](https://github.com/graphcommerce-org/graphcommerce/assets/7457340/d1c8fbe9-1827-46b6-8dcb-62ea894ddf19)

Reply:
As mentioned in the PR description
> Still requires Hygraph migration file, but is currently blocked by bug in migration cli.

[This bug](https://github.com/graphcommerce-org/graphcommerce/pull/2095) is currently fixed. I'll add the migration & docs after finishing the rest of the feedback.